samples/python/django_sample/sample_app/model.py

```diff
@@ -0,0 +1,84 @@
+''' Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+'''
+
+from django.db import models
+from django.db.models import Q, F
+
+class BaseModel(models.Model):
+  class Meta():
+    abstract = True
+  created_at = models.DateTimeField()
+  updated_at = models.DateTimeField()
+
+class Singer(models.Model):
+  class Meta():
+    db_table = 'singers'
+
+  id = models.CharField(primary_key=True, null=False)
```

Comment:
Could we move `id` into `BaseModel` as well? AFAICT, it's the same for all models.

Reply:
Since `id` is supposed to be the primary key for each of the model, moving it to the `BaseModel` is gonna cause problem as it would then work like the primary key only to the `BaseModel` and not to all the children models and Django will create another primary key Integer field for each of the children models. I tried to putting it to the base class but then insertions were not working as expected

Reply:
Ah, OK. Thanks for the explanation, that makes sense to me.